frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -1284,6 +1284,22 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 			report: this
 		});
 
+		//get rendered letterhead
+		if (print_settings.letter_head.footer) {
```

Comment:
Letterhead header can have jinja too. Have to render that as well.

Also, if we have to make a server call, can't we just use `get_print_html`? since it does `head` and `footer` rendering internally.